### ISSUE TITLE
Overriding CSS doesn't work

### ISSUE THREAD

**Issue by pomeroyb** [Radi85/Comment#219]
### Environment
* python version 3.6.1
* django Version 3.0.5
* django-comments-dab 2.6.1

### Describe the bug
I am trying to override the comment.css as described in the docs. Here is my folder layout: 
![image](https://user-images.githubusercontent.com/2469581/129494148-87dde984-88bb-4158-8c92-3c1b8637389e.png)

```
static
├── css
│   └── comment.css
└── templates
    └── comment
        └── reactions
            ├── dislike_icon_temp.html
            └── like_icon.html 
```

My custom templates work perfectly. The problem is that the default CSS still gets loaded. I am following the instructions based on the [Style Customization](https://django-comment-dab.readthedocs.io/en/latest/styling.html) docs. I feel like I'm either putting the file in the wrong place, or missing and {% load %} tag or something. 

### Expected behavior
I expect the default 'comment.css' to be overridden by my 'comment.css' in the 'static' folder.

**Comment by abhiabhi94** [comment]
Did you try to perform hard refresh(`Ctrl + Shift + R`) on the webpage, after making the changes? 

Also, I think the documentation on this front should be more clear, and should also point to the [`django`'s handling of static files](https://docs.djangoproject.com/en/dev/howto/static-files/).

Something along these lines should be more clear in my opinion.

```diff
diff --git a/docs/source/styling.rst b/docs/source/styling.rst
index acdbc7c..3bc0d5b 100644
--- a/docs/source/styling.rst
+++ b/docs/source/styling.rst
@@ -445,6 +445,9 @@ PS: The footer template is disabled by default.
 2- CSS file:
 ------------
 
-To customize the default style of comments app , you can create a ``comment.css`` file inside ``static/css`` directory.
+To customize the default style of comments app , you can create a ``comment.css`` file inside ``static/comment/css`` directory.
+This just assumes that have set ``STATIC_URL`` as ``/static/``. For more information on this, you may see `django's handling of static files`_.
+
+.. _django's handling of static files: https://docs.djangoproject.com/en/dev/howto/static-files/
 
 The new created file will override the original file used in the app.

```

Would you be willing to make a patch for this?

**Comment by pomeroyb** [comment]
I did perform a hard refresh and cleared my cache manually -- no fix.

Adding the `STATICFILES_DIRS` var to my settings.py and using the folder structure shown in the first post of this issue fixed the problem. My settings.py now has:

```
STATIC_URL = '/static/'
STATICFILES_DIRS = (
    os.path.join(BASE_DIR, 'static'),
)
```

I will note that django-comments-dab does break Django convention with regards to the static files. On the [Managing Static Files](https://docs.djangoproject.com/en/3.0/howto/static-files/) page, on section 4 of "Configuring Static Files", Django recommends to:
`Store your static files in a folder called static in your app. For example my_app/static/my_app/example.jpg.`

Django-comments-dab omits the second `my_app` folder within static, and instead uses the folder structure:
`my_app/static/example.jpg.`



So the current docs are correct, and the directory to put files in for overwriting is `static/css`.

**Comment by abhiabhi94** [comment]
> I will note that django-comments-dab does break Django convention with regards to the static files. On the Managing Static Files page, on section 4 of "Configuring Static Files", Django recommends to:
Store your static files in a folder called static in your app. For example my_app/static/my_app/example.jpg.
Django-comments-dab omits the second my_app folder within static, and instead uses the folder structure:
my_app/static/example.jpg.

Yes, this probably needs fixing, we probably try to fix this in next major version bump, `v3.0.0` to be more specific.

> So the current docs are correct, and the directory to put files in for overwriting is static/css

Where does the `static` part in here come for? I was referring to add this to the current documentation.
